Flaky TestInputsToTflintVar test

Noticed that periodically fail test `TestInputsToTflintVar`

```
=== RUN   TestInputsToTflintVar
=== PAUSE TestInputsToTflintVar
=== CONT  TestInputsToTflintVar
    tflint_test.go:38: 
        	Error Trace:	tflint_test.go:38
        	Error:      	Not equal: 
        	            	expected: []string{"--var=region=eu-central-1", "--var=instance_count=3"}
        	            	actual  : []string{"--var=instance_count=3", "--var=region=eu-central-1"}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,4 +1,4 @@
        	            	 ([]string) (len=2) {
        	            	- (string) (len=25) "--var=region=eu-central-1",
        	            	- (string) (len=22) "--var=instance_count=3"
        	            	+ (string) (len=22) "--var=instance_count=3",
        	            	+ (string) (len=25) "--var=region=eu-central-1"
        	            	 }
        	Test:       	TestInputsToTflintVar
--- FAIL: TestInputsToTflintVar (0.00s)
```

https://app.circleci.com/pipelines/github/gruntwork-io/terragrunt/1682/workflows/9247709a-4ecf-4bd2-8bab-86c9353cdbd3/jobs/10536/tests#failed-test-0

--- a/tflint/tflint_test.go
+++ b/tflint/tflint_test.go
@@ -35,6 +35,6 @@ func TestInputsToTflintVar(t *testing.T) {
 	for _, testCase := range testCases {
 		actual, err := inputsToTflintVar(testCase.inputs)
 		assert.NoError(t, err)
-		assert.Equal(t, testCase.expected, actual)
+		assert.ElementsMatch(t, testCase.expected, actual)
 	}
 }

